ChessCrafterDesktop/src/main/java/com/drewhannay/chesscrafter/panel/TeamCreationPanel.java

```diff
@@ -62,99 +62,95 @@ public void setPieceType(PieceType pieceType) {
         mTeamInfos.forEach(teamInfo -> teamInfo.label.setPiece(new Piece(teamInfo.teamId, mPieceType), teamInfo.color));
     }
 
-    private void createMouseListener(JLabel colorField, TeamInfo teamInfo) {
-        colorField.addMouseListener(new MouseListener() {
-            @Override
-            public void mouseClicked(MouseEvent e) {
-                Color c = JColorChooser.showDialog(colorField, Messages.getString("TeamCreationPanel.teamColor"), colorField.getBackground());
-                colorField.setBackground(c);
-                teamInfo.color = c;
-            }
-
-            @Override
-            public void mousePressed(MouseEvent e) {
-            }
-
-            @Override
-            public void mouseReleased(MouseEvent e) {
-            }
-
-            @Override
-            public void mouseEntered(MouseEvent e) {
-            }
-
-            @Override
-            public void mouseExited(MouseEvent e) {
-            }
-        });
-    }
-
     private void initComponents(SquareConfig squareConfig) {
         mTeamInfos.forEach(teamInfo -> {
             int offsetTeamId = teamInfo.teamId - 1;
             GridBagConstraints constraints = new GridBagConstraints();
 
             JPanel teamPanel = new JPanel();
-            teamPanel.setLayout(new GridBagLayout());
+            teamPanel.setLayout(new BoxLayout(teamPanel, BoxLayout.PAGE_AXIS));
             teamPanel.setOpaque(false);
 
-            constraints.gridy = 0;
-            constraints.insets = new Insets(0, 0, 10, 0);
-            constraints.gridwidth = 2;
-            constraints.anchor = GridBagConstraints.CENTER;
-            constraints.fill = GridBagConstraints.BOTH;
-            constraints.weightx = 1.0;
-            teamPanel.add(GuiUtility.createJLabel(Messages.getString("TeamCreationPanel.team") + teamInfo.teamId), constraints);
+            teamPanel.add(GuiUtility.createJLabel(Messages.getString("TeamCreationPanel.team") + " " + teamInfo.teamId));
```

Comment:
For this, you can change your string to:
`TeamCreationPanel.team=Team {0}`
and then use:
`Messages.getString("TeamCreationPanel.team", teamInfo.teamId)`

This will let the entire string be localized...there might be some languages that put the number first